ep/lh/ep-lh-0009.md

```diff
@@ -0,0 +1,140 @@
+# linterhub: lhman
+
+| Metadata     |                                         |
+| ------------ |-----------------------------------------|
+| EP           | ep-lh-0009                              |
+| Version      | 1                                       |
+| Title        | linterhub: lhman                        |
+| Authors      | binore                                  |
+| Status       | Draft                                   |
+
+## Abstract
+
+This document describes the `lhman` - tool that allows to request engine's catalog, install engines and prepare necessary environment.
+
+## Motivation
+
+linterhub is a catalog that contains the information necessary to install, run and process the result of the work of the `linter`. To start working, you need a utility that allows you to centrally install the linter from the existing catalog.
+
+## Proposal
+
+lhman is a utility that allows you to obtain the catalog of linters and install them. For the implementation of the functional lhman is divided into `strategies`.
+
+## Strategies
+
+lhman implements the following strategies:
+
+| Name         | Description |
+| -            | -           |
+| help         | Show help   |
+| catalog      | Get engines catalog   |
+| install      | Install engine (or multiple engines)   |
+| version      | Get lhman version|
+
+> By default, space is used as delimiter between argument and value. The equal sign is also admissible. If the argument is at the same time several values, then it is recommended to use the comma sign to separate them. In addition, multiple declaration of arguments is allowed as an alternative. Example: `lhman catalog --filter=value1,value2 --keys value1 --keys=value2`
```

Comment:
The example is not clear, you've introduced a new flag which was not defined.

Reply:
Think about another approach: `lhman install -l=linter1 -l=linter2`, just as an option to consider.